includes/class-wc-stripe-intent-controller.php

```diff
@@ -720,15 +719,15 @@ public function create_and_confirm_payment_intent( $payment_information ) {
 		$request = array_merge(
 			$request,
 			[
-				'amount'               => $payment_information['amount'],
-				'confirm'              => 'true',
-				'currency'             => $payment_information['currency'],
-				'customer'             => $payment_information['customer'],
+				'amount'                      => $payment_information['amount'],
+				'confirm'                     => 'true',
+				'currency'                    => $payment_information['currency'],
+				'customer'                    => $payment_information['customer'],
 				/* translators: 1) blog name 2) order number */
-				'description'          => sprintf( __( '%1$s - Order %2$s', 'woocommerce-gateway-stripe' ), wp_specialchars_decode( get_bloginfo( 'name' ), ENT_QUOTES ), $order->get_order_number() ),
-				'metadata'             => $payment_information['metadata'],
-				'payment_method_types' => $payment_method_types,
-				'statement_descriptor' => $payment_information['statement_descriptor'],
+				'description'                 => sprintf( __( '%1$s - Order %2$s', 'woocommerce-gateway-stripe' ), wp_specialchars_decode( get_bloginfo( 'name' ), ENT_QUOTES ), $order->get_order_number() ),
+				'metadata'                    => $payment_information['metadata'],
+				'payment_method_types'        => $payment_method_types,
+				'statement_descriptor_suffix' => $payment_information['statement_descriptor_suffix'],
```

Comment:
Does this need an `isset()` check. ie if the payment isn't a card or if the short descriptor isn't enabled, this `$payment_information['statement_descriptor_suffix']` wouldn't be set right?

Reply:
Yes :+1: